src/partials/MusicExercise.js

```diff
@@ -298,27 +272,14 @@ class MusicExercise extends React.Component {
                       TODO Tehtävät, esim: Seuraavassa tehtävässä on tarkoitus
                       opetella sointuja
                     </p>
-                    <div className="overall-container">
-                      <div className="left-container">
+                    <div class="overall-container">
```

Comment:
Reverts changes made on master to an unfixed state.

Might require reverting changes / rebasing / merging with master

Reply:
OK, now they should be reverted to a fixed state.